components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemOffer.vue

```diff
@@ -46,7 +46,7 @@
           <div
             v-else
             class="offer is-flex is-justify-content-space-evenly is-align-items-center">
-            <div>Expire In:</div>
+            <img src="/timer.svg" />
```

Comment:
I thought that we show "Expire In:" text on desktop & icon on mobile

Reply:
It's a new design for both pc and mobile